crates/modelardb_server/src/query/sorted_join_exec.rs

```diff
@@ -40,8 +41,7 @@ use datafusion::physical_plan::{
     SendableRecordBatchStream, Statistics,
 };
 use futures::stream::{Stream, StreamExt};
-
-use crate::metadata::MetadataManager;
+use modelardb_common::metadata::univariate_id_to_tag_hash;
```

Comment:
Maybe only import `metadata` and use the function as `metadata::univariate_id_to_tag_hash()` for consistency.

Reply:
Fixed.